runtime/common/src/claims.rs

```diff
@@ -391,6 +392,26 @@ decl_module! {
 			Self::process_claim(signer, who.clone())?;
 			Preclaims::<T>::remove(&who);
 		}
+
+		#[weight = (
+			T::DbWeight::get().reads_writes(4, 4) + 100_000_000_000,
```

Comment:
why such a big price on that? it's 100ms?

Reply:
not worth the time to weigh it properly - it's a root-only dispatch anyway.

Reply:
The only logic here is storage moves. The read/write weights + the base extrinsic weight should be plenty for such an extrinsic.